ibis/expr/api.py

```diff
@@ -499,6 +499,10 @@ def desc(expr: ir.Column | str) -> ir.Value:
     expr
         The expression or column name to use for sorting
 
+    See Also
+    --------
+    [`Value.desc()`](#ibis.expr.types.generic.Value.desc)
```

Comment:
In my locally built docs these seem to work because they are placed into the same .qmd file by quartodoc. Will they work differently on the published docs? I suppose making it absolute will make it easier to update the path in case we reorganize which .qmd file each item goes into, with this relative path I could see links silently breaking in the future

Reply:
Hm, yeah good point. These *should* be caught be the link checker, but the full path is also easier to understand IMO